charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -353,32 +353,32 @@ spec:
 {{ toYaml $operatorValues.spec.livenessProbe | indent 10 }}
 {{- end }}
         volumeMounts:
-        - name: reporting-operator-service-ca-bundle
-          mountPath: /var/run/configmaps/service-ca-bundle
+        - mountPath: /var/run/configmaps/service-ca-bundle
+          name: reporting-operator-service-ca-bundle
```

Comment:
@EmilyM1 Going through the [k8s volume documentation](https://kubernetes.io/docs/concepts/storage/volumes/) I don't see anything explicit in that documentation that would suggest this ordering of volume mounts is considered a best practice, but it instead appears to just be a k8s documentation standard.

Reply:
Yeah, the consistency certainly won't hurt anything though. I vote we merge it. It's structured better with order.

Reply:
This wouldn't be making it more consistent though as the [majority of charts in the repository](https://github.com/kube-reporting/metering-operator/search?q=mountPath) specify `name` as the first property in any given mount path array index.

Reply:
Clean up has to start somewhere. I volunteer to fix the rest up. Let's merge this thing, the ordering is bothering.

Reply:
Sure, and that's fine with me, but if we're making the effort to clean up these YAML files, let's do it all in a single PR rather than splitting up related changes into multiple PRs.